docs/COST_TRACKING_EVOLUTION.md

```
@@ -0,0 +1,290 @@
+browser_use_completion_tokens: int = 0     # Output tokens
+browser_use_cached_tokens: int = 0         # Cached tokens
+```
+
+---
+
+## Migration Guide
+
+### For API Consumers
+
+**Old way (estimated - removed):**
+```python
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)